spec/imsc-hrm.html

```diff
@@ -1060,6 +948,19 @@ <h3>Changes since the <a href="https://www.w3.org/TR/2021/WD-imsc-hrm-20211109/"
         </dd>
       </dl>
     </section>
+
+    <section>
+      <h3>Changes since the <a href="https://www.w3.org/TR/2023/CRD-imsc-hrm-20231220/">first Candidate Recommendation</a></h3>
+      <dl>
+        <dt>Removed support for Image Profile</dt>
+        <dd>
+          <p>Support for IMSC Image Profile, which was at feature as-risk, was removed as not sufficient implementation
+          experience could be demonstrated.</p>
```

Comment:
Reads a bit strangely, just a grammatical nit:

```suggestion
          <p>Support for IMSC Image Profile, which was an at-risk feature, was removed due to insufficient demonstrable
          implementation experience.</p>
```